agent/agentpb/common.pb.go

```diff
@@ -25,7 +24,7 @@ var _ = time.Kitchen
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
```

Comment:
So is it proto2 or proto3?

Reply:
I ask because in another recent PR I saw this go from 2->3 and naively thought it had something to do with using a slightly newer `protoc` binary. I just wanted to understand what the intent of this change is

Reply:
It has to do with the generator version and not the protoc version. I accidentally pulled v1.3.0 of protoc-gen-gofast which generated the `ProtoPackageIsVersion3` constants. We could update to v1.3.0 as the marshalers supposedly will have speed increases. However I figured when we do that we also need to update to v1.3.0 in the go.mod.

Reply:
Alrighty.